app/routes/register-your-interest/index.js

```diff
@@ -20,6 +20,9 @@ module.exports = [
     options: {
       auth: false,
       handler: async (request, h) => {
+        if (!config.registerYourInterest.enabled) {
```

Comment:
Very minor thing, but I'd switch the If statement around so that it tests for true condition not the false. 

`        if (config.registerYourInterest.enabled) {
          return h.view('register-your-interest/index', { ruralPaymentsAgency })
        }
        return h.redirect(config.urlPrefix)`